cmd/context/create.go

```diff
@@ -185,7 +185,11 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 	c.initEnvVars()
 
 	if ctxOptions.IsOkteto {
-		if err := c.initOktetoContext(ctx, ctxOptions); err != nil {
+		clusterMetadata, err := getClusterMetadata(ctx, ctxOptions.Namespace, c.OktetoClientProvider)
+		if err != nil {
+			return err
+		}
+		if err := c.initOktetoContext(ctx, ctxOptions, clusterMetadata); err != nil {
```

Comment:
could we init the context before access Okteto API in `getClusterMetadata`? Using that way if we have an invalid token we renew it and we can access metadata. I wonder if we want that or prefer to fail. You can check by removing the token from the context and running your code